cds_ils/cli.py

```diff
@@ -198,3 +241,290 @@ def run_command(command, catch_exceptions=False, verbose=True):
+        item = Item.create(i)
+        mint_record_pid(ITEM_PID_TYPE, "pid", item)
+        item.commit()
+        db.session.commit()
+        indexer.index(item)
+
+    click.secho("Items were created successfully.", fg="blue")
+
+
+@demo.command()
+@click.option("--user-email", "user_email")
+@click.option("--is-past-loan", is_flag=True)
+@with_appcontext
+def create_loan(user_email, is_past_loan):
+    """Create a loan."""
+    patron = User.query.filter_by(email=user_email).one_or_none()
+    patron_pid = patron.get_id()
+
+    loc_pid, _ = current_app_ils.get_default_location_pid
+
+    librarian_pid = (
+        User.query.filter_by(email="librarian@test.ch").one_or_none().get_id()
+    )
+
+    delivery = list(
+        current_app.config["ILS_CIRCULATION_DELIVERY_METHODS"].keys()
+    )[randint(0, 1)]
+
+    loan_dict = {
+        "pid": RecordIdProviderV2.create().pid.pid_value,
+        "patron_pid": "{}".format(patron_pid),
+        "pickup_location_pid": "{}".format(loc_pid),
+        "transaction_location_pid": "{}".format(loc_pid),
+        "transaction_user_pid": "{}".format(librarian_pid),
+        "delivery": {"method": delivery},
+    }
+
+    if is_past_loan:
+        loan_dict["state"] = "ITEM_RETURNED"
+        loan_dict["document_pid"] = "qaywb-gfe4B"
+        transaction_date = start_date = arrow.utcnow() - timedelta(days=365)
+        end_date = start_date + timedelta(days=30)
+        item_pid = "678e3-an678A"
+    else:
+        loan_dict["state"] = "ITEM_ON_LOAN"
+        loan_dict["document_pid"] = "67186-5rs9E"
+        transaction_date = start_date = arrow.utcnow() - timedelta(days=21)
+        end_date = start_date + timedelta(days=30)
+        item_pid = "vgrh9-jvj8E"
+
+    loan_dict["transaction_date"] = transaction_date.isoformat()
+    loan_dict["start_date"] = start_date.date().isoformat()
+    loan_dict["end_date"] = end_date.date().isoformat()
+    loan_dict["extension_count"] = randint(0, 3)
+    loan_dict["item_pid"] = {"type": ITEM_PID_TYPE, "value": item_pid}
+
+    loan = Loan.create(loan_dict)
+    minter(CIRCULATION_LOAN_PID_TYPE, "pid", loan)
+    loan.commit()
+    db.session.commit()
+    LoanIndexer().index(loan)
+    item = Item.get_record_by_pid(item_pid)
+    ItemIndexer().index(item)
+
+    current_search.flush_and_refresh(index="*")
+
+    doc = current_app_ils.document_record_cls.get_record_by_pid(
+        loan_dict["document_pid"]
+    )
+
+    click.secho(
+        "Loan with pid '{0}' on document '{1}' was created.".format(
+            loan_dict["pid"], doc["title"]
+        ),
+        fg="blue",
+    )
+
+
+@demo.command()
+@click.option("--user-email", "user_email")
+@click.option("--given-date", "given_date")
+@with_appcontext
+def clean_loans(user_email, given_date):
+    """Clean loans and doc requests created by given user on given date."""
+    indexer = RecordIndexer()
+
+    patron = User.query.filter_by(email=user_email).one_or_none()
```

Comment:
```suggestion
    patron = User.query.filter_by(email=user_email).one()
```
same as above, fail if not found